### DISCUSSION TITLE
Why does turning off telemetry disable AI, even local

### DISCUSSION THREAD

**Discussion by Sciumo** [wavetermdev/waveterm#720] [General]
Many people use local Ollama for AI.
Your documentation states
```
Disabling Wave AI

Interestingly, turning off telemetry automatically disables Wave AI functionality.
```
Why?

**Comment by setop** [comment]
It looks like kind of a dark pattern to me.

**Reply by axh-cmd** [reply]
Absolutely. I'll double check what if the disable telemetry really worked on the older version, will downgrade and forget about updating until they stop wanting to be the next private data hog,that will certainly leak user data like anything that's get too big and draws attention of the wrong crowd,  grateful for the open source code, disheartening that you made this choice.
